hera_mc/cm_partconnect.py

```diff
@@ -185,7 +185,7 @@ def update_part(session=None, data=None, add_new_part=False):
             session.add(part)
             session.commit()
     cm_utils.log('cm_partconn part update', data_dict=data_dict)
-    if close_session_when_done:
+    if close_session_when_done:  # pragma: no cover
```

Comment:
Why can't we cover these statements? Were we testing them before?

Reply:
We haven't been.  This is because they would use the production data, which we've avoided.  I guess we could touch those lines, but checks aren't guaranteed.

cm_partconnect has low coverage for no good reason and I was going to go through and fix that in another PR.  I can look at this issue then.